src/api.ts

```diff
@@ -0,0 +1,21 @@
+import { User, Todo, PreparedTodos } from './interfaces';
+
+export const URL_USERS = 'https://mate.academy/students-api/users';
+export const URL_TODOS = 'https://mate.academy/students-api/todos';
+
+const fetchedURL = async <T>(url: string): Promise<T[]> => {
```

Comment:
you fetch data from the API, why u named it `fetchedUrl`?
```suggestion
const fetchData = async <T>(url: string): Promise<T[]> => {
```